### PR TITLE
Translated Form assignment into French

### PR DESCRIPTION
**Pull request by LoicB**
Please find a suggestion of translation for forms assignment (#326)

### CODE REVIEW

File: 7-bank-project/2-forms/translations/assignment.fr.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Concevez votre application bancaire
```
Comment (ManuSquall) [inline 7-bank-project/2-forms/translations/assignment.fr.md:1]: What do you think about "Stylisez" instead of "Concevez"?

**Review by ManuSquall — APPROVED** [review]
Well translated 🥳🥳 thanks!

File: 7-bank-project/2-forms/translations/assignment.fr.md (hunk 1 above)
Comment (LoicB) [reply 7-bank-project/2-forms/translations/assignment.fr.md]: Yes you're right "Stylisez" is better, I corrected it.
Thanks

**Review by ManuSquall — APPROVED** [review]
LGTM 🥳🥳🥳🥳

**Pull request merged**